mkosi

```diff
@@ -1836,35 +1836,29 @@ def install_centos_old(args: CommandLineArguments,
         f.write(f"""\
 [main]
 gpgcheck=1
-
 [base]
 name=CentOS-{args.release} - Base
 {release_url}
 gpgkey={gpg_key}
-
 [updates]
 name=CentOS-{args.release} - Updates
 {updates_url}
 gpgkey={gpg_key}
-
 [extras]
 name=CentOS-{args.release} - Extras
 {extras_url}
 gpgkey={gpg_key}
-
```

Comment:
Let's keep the newlines here and below intact. We use the same style elsewhere as well so no point in removing it here.

Reply:
Sounds good, changed back to how it was